gta3/keywords.txt

```diff
@@ -2,20 +2,20 @@
 0002=goto
 0002=jump
 0003=shake_camera
-004f=create_thread
```

Comment:
@XMDS we can't rename existing keywords. It breaks existing scripts.

Reply:
@x87 I noticed that you changed "thread" to "script" earlier.  In fact, there is no such thing as "thread" in the characters defined by gta3sc.  I think this is very good because cleo itself is a scripting language.  The concept of thread can make some beginners very vague.  It turns out that these keywords exist through abbreviations.  Then it is necessary to update, we should completely remove "thread". For example:

```
{$CLEO .cs}
Thread 'Cleo'
end_thread
```
change:
```
{$CLEO .cs}
Script 'Cleo
end_script
```
In my opinion, some basic OP abbreviations are very necessary, such as jump, loop, and basic OP of the script itself.  I think we should keep a shorthand form that facilitates writing.  Actually, I think this is great

Reply:
I added new keywords while keeping old too. If you try to compile a script that uses `end_thread` it will still compile. If we remove it, it won't. And I will be getting a lot of questions onto why.

We should be consistent in our arguments. If we preserve CLEO 1.1 opcodes to [help fewer GTA3/VC scripters](https://github.com/sannybuilder/dev/issues/95#issuecomment-730463766) the same applies to old keywords.